### PR TITLE
(#439) Add verification step to QuickStart Guide

### LINKED ISSUE #439: Quickstart Guide - Add step for installation verification
## What New Or Updated Would You Like To See?

Our Quickstart Guide has been enhanced to include a suite of Pester tests that are run as a _last_ step which verifies that all previous steps completed successfully and have applied the expected configuration to the server.

## Why Is It Needed?

We need to document the additional step in the Guide which executes the tests

### PR DESCRIPTION
**Pull request by steviecoaster**
## Description Of Changes

Add documentation for verification step, which will run a suite of Pester tests against the server as the last step of the QuickStart Implementation Guide.

## Motivation and Context

We added a Pester suite of tests to the Quickstart Guide which will
verify the installation was correct.

## Testing

* [ x ] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [ x ] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

Fixes #439

### CODE REVIEW

File: input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md [hunk 1]
```diff
@@ -210,9 +210,33 @@ Below are the minimum requirements for setting up your C4B server via this guide
...
+### Step 6: Verification
+
+1. In the same **elevated** PowerShell console as above, paste and run the following code:
+
+    ```powershell
+    Set-Location "$env:SystemDrive\choco-setup\files"
+    .\Start-C4bVerification.ps1 -Fqdn '<Your expected fqdn here>'
+    ```
+
+    If you expect services to be available at `chocoserver.yourcompany.com`, then you command would look like: `.\Start-C4bVerification.ps1 -Fqdn 'chocoserver.yourcompany.com'`
```
Comment (pauby) [inline input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md:224]:
```suggestion
    If you expect services to be available at `chocoserver.yourcompany.com`, then your command would look like: `.\Start-C4bVerification.ps1 -Fqdn 'chocoserver.yourcompany.com'`

```

**Review by pauby — CHANGES_REQUESTED** [review]

File: input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md (hunk 1 above)
Comment (steviecoaster) [reply input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md]: Fixed!

**Review by pauby — APPROVED** [review]

**Review by ryanrichter94 — COMMENTED** [review]
Found the following when running the Test suite on a new QSG deployment.

**Review by gep13 — APPROVED** [review]
LGTM!

**Pull request merged**